Source/TransportSession/UnauthenticatedTransportSession.swift

```diff
@@ -77,21 +80,36 @@ final public class UnauthenticatedTransportSession: NSObject, UnauthenticatedTra
     private let userAgent: ZMUserAgent
     public var environment: BackendEnvironmentProvider
     fileprivate let reachability: ReachabilityProvider
-    
+    /// Property to accept requests
+    public let ready: Bool
```

Comment:
Suggestion to rename it for better clarity

```suggestion
    public let readyForRequests: Bool
```